bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -103,6 +103,12 @@ public void textBoxKeyPress(final TextBoxWidget tbwe) {
         });
 
         _agsPub.subscribe((obs, oldOptions, newOptions) -> updateGuiding());
+
+        BagsManager.addBagsStatusListener((key, oldStatus, newStatus) -> {
+            // Need to compare by reference here.
+            if (key.equals(getContextObservation().getNodeKey()))
+                updateGuiding();
```

Comment:
I wonder if `updateGuiding` is overly broad in that it also configures other widgets.  The state change event can happen at any time, even while editing some other control which would then be reset?  Perhaps it would make sense to separate the feedback update into its own method.